action.yaml

```diff
@@ -30,8 +30,13 @@ runs:
       id: compose-args
       shell: bash
       run: |
-        encoded_github="$(echo '${{ inputs.github_context }}' | base64 -w 0)"
-        encoded_runner="$(echo '${{ inputs.runner_context }}' | base64 -w 0)"
+        echo "${{ inputs.github_context }}" > github_context.json
```

Comment:
How will this work with the `"` in the json? Does it properly escape them or will it early close this echo string?

Reply:
Work with actual bash variables now... see passing builds below